srcpkgs/c-ares/template

```diff
@@ -1,22 +1,19 @@
 # Template file for 'c-ares'
 pkgname=c-ares
-version=1.16.1
+version=1.17.1
 revision=1
 build_style=gnu-configure
 short_desc="C library that performs DNS requests and name resolves asynchronously"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://c-ares.haxx.se/"
-distfiles="https://c-ares.haxx.se/download/${pkgname}-${version}.tar.gz"
-checksum=d08312d0ecc3bd48eee0a4cc0d2137c9f194e0a28de2028928c0f6cae85f86ce
+changelog="${homepage}/changelog.html"
```

Comment:
Put the link without any variables, it's good to be able to copy and paste it directly from the template into a browser.

Reply:
It crossed my mind but I wasn't sure if anyone is accessing this "manually" from the template (vs. xbps-query or xbps-src), but sounds reasonable.

BTW it may worth codifying those kind of stylistic preferences within the manual, as well as xlint.

thanks for the reviews!

Reply:
Indeed, it would make sense. Feel free to open a PR / issue about it, otherwise I will try to get to it.